include/z64actor.h

```diff
@@ -1183,4 +1183,98 @@ typedef enum {
     /* 0xC8 */ CLEAR_TAG_SMOKE = 200
 } ClearTagType;
 
+typedef enum {
+    /* 0x00 */ TATL_HINT_ID_DEFAULT,
+    /* 0x01 */ TATL_HINT_ID_SNAPPER,
+    /* 0x02 */ TATL_HINT_ID_MINI_BABA,
+    /* 0x03 */ TATL_HINT_ID_MAD_JELLY,
+    /* 0x04 */ TATL_HINT_ID_SKULLTULA,
+    /* 0x05 */ TATL_HINT_ID_RED_CHUCHU,
+    /* 0x06 */ TATL_HINT_ID_BLUE_CHUCHU,
+    /* 0x07 */ TATL_HINT_ID_DEKU_BABA,
+    /* 0x08 */ TATL_HINT_ID_BIO_DEKU_BABA,
+    /* 0x09 */ TATL_HINT_ID_WILTED_DEKU_BABA,
+    /* 0x0A */ TATL_HINT_ID_DEXIHAND,
+    /* 0x0B */ TATL_HINT_ID_NEJIRON,
+    /* 0x0C */ TATL_HINT_ID_GIANT_BEE,
+    /* 0x0D */ TATL_HINT_ID_DODONGO,
+    /* 0x0E */ TATL_HINT_ID_DEEP_PYTHON,
+    /* 0x0F */ TATL_HINT_ID_DEATH_ARMOS,
+    /* 0x10 */ TATL_HINT_ID_DINOLFOS,
+    /* 0x11 */ TATL_HINT_ID_FIRE_KEESE,
+    /* 0x12 */ TATL_HINT_ID_KEESE,
+    /* 0x13 */ TATL_HINT_ID_ARMOS,
+    /* 0x14 */ TATL_HINT_ID_EENO,
+    /* 0x15 */ TATL_HINT_ID_DRAGONFLY,
+    /* 0x16 */ TATL_HINT_ID_IGOS_DU_IKANA,
+    /* 0x17 */ TATL_HINT_ID_GARO,
+    /* 0x18 */ TATL_HINT_ID_GARO_MASTER,
+    /* 0x19 */ TATL_HINT_ID_WART,
+    /* 0x1A */ TATL_HINT_ID_GOMESS,
+    /* 0x1B */ TATL_HINT_ID_GOHT,
+    /* 0x1C */ TATL_HINT_ID_BLUE_BUBBLE,
+    /* 0x1D */ TATL_HINT_ID_KINGS_LACKEYS_DARK_ROOM,
+    /* 0x1E */ TATL_HINT_ID_KINGS_LACKEYS_LIGHT_ROOM,
+    /* 0x1F */ TATL_HINT_ID_SKULLWALLTULA,
+    /* 0x20 */ TATL_HINT_ID_GOLD_SKULLTULA, // Only in JP Ver.
+    /* 0x21 */ TATL_HINT_ID_CAPTAIN_KEETA,
+    /* 0x22 */ TATL_HINT_ID_YELLOW_CHUCHU,
+    /* 0x23 */ TATL_HINT_ID_TAKKURI,
+    /* 0x24 */ TATL_HINT_ID_RED_BUBBLE,
+    /* 0x25 */ TATL_HINT_ID_HIPLOOP,
+    /* 0x26 */ TATL_HINT_ID_MASKED_HIPLOOP,
+    /* 0x27 */ TATL_HINT_ID_SHELLBLADE,
+    /* 0x28 */ TATL_HINT_ID_GYORG, // "if you go near, you'll be eaten!"
+    /* 0x29 */ TATL_HINT_ID_GYORG_STUNNED, // "Jump in and attack it!"
+    /* 0x2A */ TATL_HINT_ID_REDEAD,
+    /* 0x2C */ TATL_HINT_ID_BLACK_BOE = 0x2C,
+    /* 0x2D */ TATL_HINT_ID_GIBDO,
+    /* 0x2E */ TATL_HINT_ID_TWINMOLD,
+    /* 0x2F */ TATL_HINT_ID_WEARING_GIANTS_MASK, // Twinmold, JP Ver. only
+    /* 0x30 */ TATL_HINT_ID_WALLMASTER,
+    /* 0x31 */ TATL_HINT_ID_FLOORMASTER,
+    /* 0x32 */ TATL_HINT_ID_MAJORAS_MASK,
+    /* 0x33 */ TATL_HINT_ID_MAJORAS_INCARNATION,
+    /* 0x34 */ TATL_HINT_ID_MAJORAS_WRATH,
+    /* 0x35 */ TATL_HINT_ID_IRON_KNUCKLE,
+    /* 0x37 */ TATL_HINT_ID_LIKE_LIKE = 0x37,
+    /* 0x39 */ TATL_HINT_ID_BEAMOS = 0x39,
```

Comment:
It does seem like if you're going to have one empty one, you might as well fill out the other few.